projects/eslint-config/plugins/portal/tests/lib/rules/no-react-dom-create-portal.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const MultiTester = require('../../../../../scripts/MultiTester');
+const rule = require('../../../lib/rules/no-react-dom-create-portal');
+
+const parserOptions = {
+	ecmaFeatures: {
```

Comment:
Isn't ecma an abbreviation? Wouldn't `ECMA` be more appropriate?

Reply:
This is an option passed to babel-eslint, so we can't update this key

Reply:
In any case, when it is at the start of the word, we usually lowercase it (at least in portal...)